landoapi/phabricator.py

```diff
@@ -348,19 +350,21 @@ class PhabricatorSubsystem(Subsystem):
     name = "phabricator"
 
     def ready(self):
-        if (
-            self.flask_app.config["PHABRICATOR_UNPRIVILEGED_API_KEY"]
-            and re.match(
-                r"^api-.{28}$",
-                self.flask_app.config["PHABRICATOR_UNPRIVILEGED_API_KEY"],
-            )
-            is None
-        ):
+        unpriv_key = self.flask_app.config["PHABRICATOR_UNPRIVILEGED_API_KEY"]
+        priv_key = self.flask_app.config["PHABRICATOR_ADMIN_API_KEY"]
+
+        if unpriv_key and PHAB_API_KEY_RE.match(unpriv_key) is None:
```

Comment:
you probably want `re.search()` not `re.match()` - your regex has a start of string anchor making `match` redundant, and imho `search` is clearer